core/embed/bootloader/header.S

```diff
@@ -20,7 +24,10 @@ g_header:
     .byte FIX_VERSION_MINOR          // fix_vminor
     .byte FIX_VERSION_PATCH          // fix_vpatch
     .byte FIX_VERSION_BUILD          // fix_vbuild
-    . = . + 8                        // reserved
+    .word HW_MODEL                   // type of the designated hardware
+    .byte HW_REVISION                // revision of the designated hardware
+    .byte VERSION_MONOTONIC          // monotonic version
```

Comment:
Monotonic version of what? Of the hardware?

Reply:
of the bootloader

Reply:
in that case please change the comment to `monotonic version of the bootloader`

Reply:
i sort of don't want to do that:
- the `VERSION_xx` bytes don't comment this either
- this is also present in other headers (fw, reflash, prodtest), and adding the comment would cause unnecessary diff between the header files

Though i get that inserting the `HW_MODEL` between the `FIX_VERSION` and `VERSION_MONOTONIC` could be confusing (did this for alignment reasons), maybe something like: `monotonic version of the binary` so that there is no diff

Reply:
OK

Reply:
done here: https://github.com/trezor/trezor-firmware/pull/2649/commits/83fa07158a7c9ff7722740d1ec76b098e71982e6